src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -713,6 +728,7 @@ void CCC_Register()
 
     // Render device states
     CMD4(CCC_Integer, "r__supersample", &ps_r__Supersample, 1, 4);
+    CMD2(CCC_SunDirection, "r__sun_direction", &ps_sun_direction);
```

Comment:
I think this should be an `rs_` command, `r__supersample` seems to be an outlier here.